templates/repo/diff/comments.tmpl

```diff
@@ -3,7 +3,7 @@
 {{$createdStr:= TimeSinceUnix .CreatedUnix ctx.Locale}}
 <div class="comment" id="{{.HashTag}}">
 	{{if .OriginalAuthor}}
-		<span class="avatar"><img src="{{AppSubUrl}}/assets/img/avatar_default.png"></span>
+		<span class="avatar">{{ctx.AvatarUtils.AvatarDefault}}</span>
```

Comment:
At first, I did as you said. But for other maintainers. they may consider where does this `28` come from?
Maybe we can merge this in to `AvatarUtils.Avatar`, set `item` to be `nil` or something else?

Reply:
I'm in favor of the helper function. We use helper functions for all other avatars, why not this one too?

Reply:
The real situation here is:

* There are still not only one `avatar_default.png` usages in JS code, not in tmpl, so your new "helper function" won't cover them.
* If we'd like to improve the "default avatar" in the future, developers still to search the whole code base, don't forget the JS ones.
* Using `<img class="ui avatar gt-vm" ` is also readable and it is standard HTML. The `28` is the default width, even if you use a helper function, in many uses you still need to write the side (you see, you use 40 now ..... you could also aski "where it comes from", the same)


So, introducing a helper function here seems an over-engineering IMO:

1. There are only a few (2?) cases, not too many, not reduce duplication really.
1. It doesn't cover all cases, there will be still not only one `avatar_default.png` in other places. Developers still need to search `avatar_default.png` to avoid missing anything.
1. It needs more lines of code, but doesn't really help to improve the readability or maintainability, standard HTML code is also very readable and maintainable.
1. The current approach introduces garbage code like `<span class="avatar">`


Instead of introducing a new helper function, the new commit which uses existing helper function seems better.

Reply:
> you see, you use 40 now

It comes from:
![image](https://github.com/go-gitea/gitea/assets/18380374/6980219e-5d44-47dd-b5e1-f0cf95e2a518)
At least for me, it can be easily considered that the avatars should be 40 here, but not the default size.